services/QuillLMS/app/workers/test_for_earned_checkboxes_worker.rb

```diff
@@ -6,8 +6,8 @@ def perform(id)
     teacher = User.find_by(id: id)
     if teacher
```

Comment:
To remove the larger conditional, I would add a guard clause:

`return unless teacher`